force-app/main/settingsComponentLibrary/lwc/settingsRowInput/settingsRowInput.js

```diff
@@ -0,0 +1,57 @@
+import { LightningElement, api } from "lwc";
+
+export default class SettingsRowInput extends LightningElement {
+    @api title;
+    @api description;
+    @api inputId;
+    @api value;
+    @api placeholder;
+    @api options;
+    @api disabled;
+    @api type;
+
+    handleInputChange(event) {
+        let eventDetail = {};
+
+        if (this.type === "toggle") {
```

Comment:
Please change this to a switch statement to match the other conditionals we have below.